### PR TITLE
fix: DEV-3151: Add simple compare eval item

### PR DESCRIPTION
**Pull request by KonstantinKorotaev**

### CODE REVIEW

**Review by triklozoid — APPROVED** [review]

File: evalme/classification.py [hunk 1]
```diff
@@ -30,16 +30,19 @@ def exact_match(self, item, label_weights=None, per_label=False):
...
                 else:
                     return 0
             if not isinstance(labels, list):
                 labels = [labels]
-
             if not isinstance(y_labels, list):
                 y_labels = [y_labels]
+            # Check if spans are mismatched
+            mismatched_spans = False
+            if EvalItem.has_spans([x, y]):
```
Comment (niklub) [inline evalme/classification.py:42]: it's not ideal solution - what if choices will be attached to other annotations, boxes, polygons, etc. ?

File: evalme/eval_item.py [hunk 2]
```diff
@@ -31,3 +31,139 @@ def empty(self):
...
+    @staticmethod
+    def bbox_iou(boxA, boxB):
+        """
+        Intersection over union for bbox with x/y/width/height
+        :param boxA:
+        :param boxB:
+        :return:
+        """
+        # check data
+        assert boxA['x']
```
Comment (niklub) [inline evalme/eval_item.py:71]: that gives an error when boxA['x'] == 0 and exception with non-existed key

**Review by niklub — CHANGES_REQUESTED** [review]

**Comment by niklub** [comment]
I can't understand - why can't we simply reuse the same hack made for https://github.com/heartexlabs/label-studio-evalme/blob/master/evalme/text/text.py#L35 ?

**Comment by KonstantinKorotaev** [comment]
> I can't understand - why can't we simply reuse the same hack made for https://github.com/heartexlabs/label-studio-evalme/blob/master/evalme/text/text.py#L35 ?

We can do that, but there will code duplication.
This PR presents EvalItem.max_score which incapsulates logic from previous [solution](https://github.com/heartexlabs/label-studio-evalme/blob/master/evalme/text/text.py#L59). It can be reused in other methods without duplication.

File: evalme/eval_item.py (hunk 2 above)
Comment (KonstantinKorotaev) [reply evalme/eval_item.py]: Fixed

**Review by niklub — APPROVED** [review]

**Pull request merged**